src/terra-dev-site/snt/secondary scroll test/Themed.module.scss

```diff
@@ -0,0 +1,13 @@
+:local {
```

Comment:
can we reuse the Themed.module.scss file provided at `scr/terra-dev-site/snt/` instead of creating a new one?
https://github.com/cerner/terra-dev-site/blob/master/src/terra-dev-site/snt/Themed.module.scss

Reply:
Agreed. I removed the duplicate stylesheet: https://github.com/cerner/terra-dev-site/pull/217/commits/d455c08249a5d437538211b2b05f9bdef7e4ff0b